code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -513,6 +513,11 @@
 	if(target.check_block())
 		target.visible_message("<span class='warning'>[target] blocks [user]'s disarm attempt!</span>")
 		return FALSE
+	if(target.absorb_stun(0))
+		target.visible_message("<span class='warning'>[target] is not affected by [user]'s disarm attempt!</span>")
+		user.do_attack_animation(target, ATTACK_EFFECT_DISARM)
```

Comment:
Not sure about the animation if they absorbed the stun.

Reply:
If you get pushed but you are not phased by it, it still looks like you got pushed

Reply:
I've tested both and it imo looks much better animated. Like hal said, they are still trying to push you.